src/open_inwoner/openzaak/cases.py

```diff
@@ -16,7 +18,7 @@
 
 
 # cache for 3 minutes to quickly switch between open and closed cases
```

Comment:
Maybe confusing since this is modified by the settings now. We can add that the default value is 3 minutes. If you think it's excessive we can leave it as is.

Reply:
Yes you are right, the comment doesn't have to be there. I appended a commit to remove it.